src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -4,7 +4,7 @@ import {StyleSheet, Text, View} from 'react-native';
 
 import dimen from '../../utils/dimen';
 import {COLORS} from '../../utils/theme';
-import {SIGNED} from '../../hooks/core/constant';
+import {ANONYMOUS, SIGNED} from '../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of the `ANONYMOUS` constant seems to be a new addition. Ensure that this constant is properly defined and exported in the `../../hooks/core/constant` module. If it's not used elsewhere, consider defining it directly in this file to reduce dependencies.
```diff
- import {ANONYMOUS, SIGNED} from '../../hooks/core/constant';
+ const ANONYMOUS = 'Anonymous'; // or whatever the value should be
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->